src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -682,4 +691,27 @@ Run 'dotnet {1} --show-aliases' with no args to show all aliases.</value>
   <data name="ListTemplatesCommand" xml:space="preserve">
     <value>To list installed templates, run 'dotnet new --list'.</value>
   </data>
-</root>
+  <data name="ArgumentsSectionTitle" xml:space="preserve">
+    <value>Arguments:</value>
+  </data>
+  <data name="CommandUsage" xml:space="preserve">
+    <value>Usage: 
```

Comment:
@KathleenDollard 
I put the usage as below. Note: it is not similar to other dotnet tools, however it is more descriptive for user. 

https://docs.microsoft.com/dotnet/core/tools/dotnet-new uses similar way.